client/post-editor/media-modal/image-editor/image-editor-canvas.jsx

```diff
@@ -97,12 +100,22 @@ const MediaModalImageEditorCanvas = React.createClass( {
 
 		this.drawImage();
 		this.updateCanvasPosition();
+		this.onWindowResize = throttle( this.updateCanvasPosition, 200 );
+		if ( window ) {
```

Comment:
This will throw an error if window isn't defined. If we'd like to do a check a safer one would be:

```javascript
if ( typeof window !== 'undefined' )
```